include/hobbes/net.H

```diff
@@ -324,7 +324,7 @@ _HNET_DEFINE_PRIMTYS(int32_t,  "int");
 _HNET_DEFINE_PRIMTYS(uint32_t, "int");
 _HNET_DEFINE_PRIMTYS(int64_t,  "long");
 _HNET_DEFINE_PRIMTYS(uint64_t, "long");
-_HNET_DEFINE_PRIMTYS(size_t,   "long");
+//_HNET_DEFINE_PRIMTYS(size_t,   "long");
```

Comment:
I don't have clock_gettime issue as well. here is my compiler info
Apple LLVM version 8.1.0 (clang-802.0.42)
Target: x86_64-apple-darwin16.6.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/Xcode